.yamllint

```diff
@@ -1,6 +1,10 @@
 ---
 extends: default
 rules:
+  document-start: disable
```

Comment:
same here - is it really necessary to change this?

Reply:
see above (also these document start dashes are absolutely pointless)